network/per_host_subnet.go

```diff
@@ -0,0 +1,64 @@
+package network
+
+import (
+	"encoding/json"
+	"strings"
+
+	simplejson "github.com/bitly/go-simplejson"
+	"github.com/pkg/errors"
+	"github.com/rancher/go-rancher-metadata/metadata"
+)
+
+const (
+	PerHostSubnetKey = "__host_label__"
```

Comment:
This keyword is not related to per host subnet. It can be used with anything else. It's just a keyword to specify we want the value to be replaced with the value of the host label.